xds/internal/balancer/edsbalancer/eds_impl_test.go

```diff
@@ -19,30 +19,30 @@
 package edsbalancer
 
 import (
+	"context"
+	"errors"
 	"fmt"
-	"reflect"
 	"sort"
 	"testing"
 	"time"
 
 	corepb "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
 	"github.com/google/go-cmp/cmp"
-	"github.com/google/go-cmp/cmp/cmpopts"
-	"google.golang.org/grpc/internal"
-	"google.golang.org/grpc/xds/internal/balancer/loadstore"
-
 	"google.golang.org/grpc/balancer"
 	"google.golang.org/grpc/balancer/roundrobin"
 	"google.golang.org/grpc/connectivity"
 	"google.golang.org/grpc/internal/balancer/stub"
-	"google.golang.org/grpc/internal/xds/env"
-	xdsinternal "google.golang.org/grpc/xds/internal"
+	"google.golang.org/grpc/resolver"
 	"google.golang.org/grpc/xds/internal/balancer/balancergroup"
+	"google.golang.org/grpc/xds/internal/balancer/clusterimpl"
+	"google.golang.org/grpc/xds/internal/balancer/weightedtarget"
 	"google.golang.org/grpc/xds/internal/testutils"
+	"google.golang.org/grpc/xds/internal/testutils/fakeclient"
 	"google.golang.org/grpc/xds/internal/xdsclient"
-	"google.golang.org/grpc/xds/internal/xdsclient/load"
 )
 
+var errAllPrioritiesRemoved = errors.New("no locality is provided, all priorities are removed")
```

Comment:
Could we export this error from the `priority` package and use that instead of defining one here to mirror the former?

Reply:
Done